### PR TITLE
Migrate from SSR to React

### PR DESCRIPTION
**Pull request by tristanoneil**
In order to make building out new API features a bit more straight forward and [less like this](https://github.com/calendly/buzzwordcrm/blob/main/views/events.hbs#L36). This refactors SSR express views using HBS to React and React Router. This PR uses [ESBuild](https://esbuild.github.io) in order to bundle client-side JS.

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -2,19 +2,43 @@ const express = require('express');
...
-    const {
-        collection,
-        pagination
-    } = await calendlyService.getUserScheduledEvents(user, count, page_token);
-    const events = collection.map(formatEventDateTime);
+        const {
+            collection,
+            pagination
+        } = await calendlyService.getUserScheduledEvents(calendly_uid, count, page_token);
+        const events = collection.map(formatEventDateTime);
```
Comment (RSalaam) [inline routes/api.js:17]: Looks as if there are some spacing issues! Hopefully, it's not too much of a pain to get rid of.

Comment (tristanoneil) [reply routes/api.js]: I added/ran Prettier in the last two commits so we don't have to worry about this going forward.

Comment (RSalaam) [reply routes/api.js]: Beautiful! Thank you.

File: calendlyOauth2Strategy/index.js [hunk 2]
```diff
@@ -1,37 +1,33 @@
...
 const User = require('../models/userModel');
 const CalendlyService = require('../services/calendlyService');
-const {
-    CALENDLY_AUTH_BASE_URL,
-    CLIENT_ID,
-    CLIENT_SECRET,
-    REDIRECT_URI
-} = process.env;
+const { CALENDLY_AUTH_BASE_URL, CLIENT_ID, CLIENT_SECRET, REDIRECT_URI } =
+  process.env;
```
Comment (RSalaam) [inline calendlyOauth2Strategy/index.js:5]: It's interesting that this has been condensed into a single line with prettier!

**Review by RSalaam — APPROVED** [review]
Thank you for getting all of this set up! Approved. You'll see some comments, but nothing major.

**Pull request merged**